src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -634,14 +634,34 @@ public static void ParseExact_String_String_FormatProvider_DateTimeStyles_G()
             Assert.Equal(expectedString, result.ToString("g"));
         }
 
-        [Fact]
-        public static void ParseExact_String_String_FormatProvider_DateTimeStyles_O()
+        [Theory]
+        [MemberData("Format_String_TestData_O")]
```

Comment:
@justinvp , Thanks for spotting that. I don't know how I missed that as the pattern we use.